### PR TITLE
Generate code using ack runtime v0.13.0 and aws-sdk-go v1.38.52

### PR DESCRIPTION
**Pull request by kumargauravsharma**
Issue #, if available:

This PR generates ACK ElastiCache controller code using runtime v0.13.0, latest code-generator and aws-sdk-go v1.38.52

Description of changes:
* Updated go.mod to use `github.com/aws-controllers-k8s/runtime` `v0.13.0`
* Updated sdk template per current signature of `GoCodeSetCreateOutput`
* Regenerated code using latest `code-generator`
* Fixed unit tests per latest fields in Spec and updated Conditions error message format
* Enhanced logging when unit test fails

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by A-Hilaly — APPROVED** [review]
:+1:

File: pkg/resource/replication_group/testdata/replication_group/cr/rg_cmd_create_initiated.yaml [hunk 1]
```diff
@@ -8,6 +8,10 @@ spec:
   replicasPerNodeGroup: 1
   replicationGroupDescription: cluster-mode disabled RG
   replicationGroupID: rg-cmd
+  atRestEncryptionEnabled: false
+  SnapshotRetentionLimit: 0
+  snapshotWindow: "09:00-10:00"
+  TransitEncryptionEnabled: false
```
Comment (jaypipes) [inline pkg/resource/replication_group/testdata/replication_group/cr/rg_cmd_create_initiated.yaml:14]: not sure it matters, but `TransitEncryptionEnabled` should be `transitEncryptionEnabled` here and in the similar files below.

**Review by jaypipes — APPROVED** [review]
:+1: thanks @kumargauravsharma!

**Pull request merged**